<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CamMetWidget</class>
 <widget class="QWidget" name="C_CamMetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_CamMetWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgeWiSectionHeader" name="pc_WiTitle" native="true"/>
   </item>
   <item>
    <widget class="stw::opensyde_gui::C_CamMetControlBarWidget" name="pc_ControlWidget" native="true"/>
   </item>
   <item>
    <widget class="stw::opensyde_gui::C_CamMetTreeView" name="pc_TraceView">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw::opensyde_gui::C_CamMetStatusBarWidget" name="pc_StatusWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui::C_CamMetTreeView</class>
   <extends>QTreeView</extends>
   <header>C_CamMetTreeView.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_CamMetControlBarWidget</class>
   <extends>QWidget</extends>
   <header>C_CamMetControlBarWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_CamMetStatusBarWidget</class>
   <extends>QWidget</extends>
   <header>C_CamMetStatusBarWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgeWiSectionHeader</class>
   <extends>QWidget</extends>
   <header>C_CamOgeWiSectionHeader.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
